course/tests/tests_exportcourse.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management import CommandError, find_commands
+from django.test import TestCase
+from django.core import management
+from io import StringIO
+import sys
+
+
+class CommandTests(TestCase):
+    databases = '__all__'
+    fixtures = ["dumps/courseData.json"]
+
+    def test_exportcourse(self):
+        sys.stdout = out = StringIO()
+        management.call_command('exportcourse', 1, stdout=out, verbosity=3)
+        self.assertIn("Exporting course Spanish for English speakers course", out.getvalue())
+
+    def test_calling_a_command_without_argument(self):
```

Comment:
actually, this one is also Django built in functionality, so we don't need it.

The other 2 are correct now, so just remove this and I'll merge :smile_cat:

Reply:
Ah okay